### PR TITLE
Grant assets dir more relaxing permission

### LINKED ISSUE #22: /model/predict endpoints fails on Red Hat OpenShift due to permission error
```

INFO:tensorflow:Loading checkpoint /workspace/assets/model-238410
--
  | Traceback (most recent call last):
  | File "core/getpoint/run_summarization.py", line 334, in <module>
  | tf.app.run()
  | File "/opt/conda/lib/python3.6/site-packages/tensorflow/python/platform/app.py", line 44, in run
  | _sys.exit(main(_sys.argv[:1] + flags_passthrough))
  | File "core/getpoint/run_summarization.py", line 327, in main
  | decoder = BeamSearchDecoder(model, batcher, vocab)
  | File "/workspace/core/getpoint/decode.py", line 69, in __init__
  | if not os.path.exists(self._decode_dir): os.mkdir(self._decode_dir)
  | PermissionError: [Errno 13] Permission denied: '/workspace/assets/decode'


```

### PR DESCRIPTION
**Pull request by xuhdev**
Fixes #22

### CODE REVIEW

**Comment by xuhdev** [comment]
Is there any way we can test this?

**Comment by autoih** [comment]
> Is there any way we can test this?

@ptitzler

File: Dockerfile [hunk 1]
```diff
@@ -33,6 +33,8 @@ COPY . /workspace
 # check file integrity
 RUN md5sum -c md5sums.txt
 
+RUN chmod 777 ./assets
+
```
Comment (autoih) [inline Dockerfile:37]: Do we need to allow the permission to all?

**Comment by xuhdev** [comment]
Closed in favor of #24

**Pull request closed without merging**